datalad/distributed/create_sibling_gin.py

```diff
@@ -0,0 +1,133 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""High-level interface for creating a publication target on a GIN instance
+"""
+
+import logging
+
+from datalad.distributed.create_sibling_ghlike import _create_sibling
+from datalad.distributed.create_sibling_gogs import _GOGS
+from datalad.distribution.dataset import datasetmethod
+from datalad.interface.base import (
+    Interface,
+    build_doc,
+)
+from datalad.interface.utils import eval_results
+
+lgr = logging.getLogger('datalad.distributed.create_sibling_gin')
+
+
+class _GIN(_GOGS):
+    """Customizations for GIN as a GH-like platform
+    """
+    name = 'gin'
+    fullname = 'GIN'
+    response_code_unauthorized = 401
+
+    def normalize_repo_properties(self, response):
+        """Normalize the essential response properties for the result record
+        """
+        return dict(
+            reponame=response.get('name'),
+            private=response.get('private'),
+            # GIN reports the SSH URL as 'clone_url', but we need
+            # a HTML URL (without .git suffix) for setting up a
+            # type-git special remote (if desired)
+            clone_url=response.get('html_url'),
+            ssh_url=response.get('ssh_url'),
+            html_url=response.get('html_url'),
```

Comment:
Saw that in a couple of spots, I think, and I'm a bit confused. What's an HTML-URL? Shouldn't that be an HTTP-URL? If it's a about a browsable site to report on, that's okay, but the comment reads we need such to setup git type remote, so possible HTML content at the URL doesn't seem to be what we are after here.